neuvector-scanner.yaml

```diff
@@ -1,22 +1,24 @@
 package:
   name: neuvector-scanner
   version: "3.875"
-  epoch: 0
+  epoch: 1
   description: NeuVector vulnerability scanner for the SUSE NeuVector Container Security Platform
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
       - ca-certificates
-      - neuvector-db
```

Comment:
This shouldn't be removed: we still need to pull in the subpackage when the main `-scanner` is installed.